### PR TITLE
Editor friendly way to comment, spell check, etc on docker-compose file

### PR DESCRIPTION
**Pull request by mevinbabuc**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -45,25 +45,24 @@ services:
...
-                
-            
-            
+
         ports:
             - '8000:8000'
         depends_on:
             - postgres
-            # - influxdb
+#             - influxdb
```
Comment (dabeeeenster) [inline docker-compose.yml:57]: Not sure the reason for the indentation here?

File: docker-compose.yml [hunk 2]
```diff
@@ -45,25 +45,24 @@ services:
             # EMAIL_HOST_PASSWORD: smtp_account_password
             # EMAIL_PORT: 587 # optional
             # EMAIL_USE_TLS: True # optional
-            
-            # ENABLE_ADMIN_ACCESS_USER_PASS: True # set to True to enable access to the /admin/ Django backend via your username and password
+
+            # set to True to enable access to the /admin/ Django backend via your username and password
+            ENABLE_ADMIN_ACCESS_USER_PASS: 'true'
```
Comment (dabeeeenster) [inline docker-compose.yml:50]: This is disabled by default for additional security. What's your reason for enabling it?

**Review by dabeeeenster — COMMENTED** [review]
Thanks for this! Just a couple of questions!

**Comment by matthewelwell** [comment]
Sorry @mevinbabuc, I'm going to close this one since we haven't had any feedback on the comments that @dabeeeenster left. Feel free to reopen at a later date.

**Pull request closed without merging**